newIDE/app/src/EventsSheet/index.js

```diff
@@ -169,6 +169,7 @@ export default class EventsSheet extends React.Component<Props, State> {
   _eventSearcher: ?EventsSearcher;
   _searchPanel: ?SearchPanel;
   _containerDiv = React.createRef<HTMLDivElement>();
+  _zoomLevel = {min: 1, max: 50};
```

Comment:
uber nit: if these are just constant values I would rather store them outside of the component (to keep the interface of the component as lean as possible and show that they won't ever be changed according to the component or something like this)

Reply:
Good idea